doc/release-notes/release-4.sgml

```diff
@@ -189,6 +190,12 @@ Most user-facing changes are reflected in squid.conf (see below).
    the OpenSSL options when using GnuTLS.
 
 
+<sect1>ESI Custom Parser removal
+<p>The Squid custom ESI parser has been removed in favour of better supported
+and maintained third-party libraries. At least one of libxml2 or libexpat are
```

Comment:
s/are/is/ or, better, rephrase as "To enable ESI support, libxml2 or libexpat is now required." and spell out "(Edge Side Includes)" after the _first_ mentioning of the term in this paragraph.